coderd/templates.go

```diff
@@ -658,7 +677,8 @@ func (api *API) patchTemplateMeta(rw http.ResponseWriter, r *http.Request) {
 			inactivityTTL != time.Duration(template.TimeTilDormant) ||
 			timeTilDormantAutoDelete != time.Duration(template.TimeTilDormantAutoDelete) ||
 			req.AllowUserAutostart != template.AllowUserAutostart ||
-			req.AllowUserAutostop != template.AllowUserAutostop {
+			req.AllowUserAutostop != template.AllowUserAutostop ||
+			req.RequireActiveVersion != template.RequireActiveVersion {
```

Comment:
I don't think this check is required any more, since schedule store doesn't set this field.